packages/radio/src/ux-radio.ts

```diff
@@ -182,7 +182,7 @@ export class UxRadio implements UxComponent {
       winEvents.subscribe('mouseup', upAction, true);
     }
 
-    return true;
+    e.preventDefault();
```

Comment:
doesn't returning `false`/`preventDefault` prevent the radio from working normally?

Reply:
I need to be honest: I'm not sure :-/ 

I have tried `return false`, `return true`, `preventDefault()` and everything work. I think I choose the `preventDefault` as it was what was in the `ux-checkbox` similar method. But in this case I'm not sure what is the best thing to do ?

Reply:
>  and everything work.

lol. I think for now, lets return `true`, unless we are sure this mouse down isn't supposed to toggle the radio via native behavior (mousedown on label or the radio itself).

Reply:
Ok, will do the same in checkbox and we can follow this closely in the near future.

Are you OK if I place a `// TODO:` here as a follow-up ?

Reply:
let's do it